docs/howto/system_testing.md

```diff
@@ -402,6 +409,15 @@ for system tests.
 
 | Option | Type | Required | Description |
 |---|---|---|---|
+| agent.linux_capabilities | array string | | Linux Capabilities that must be enabled in the system to run the Elastic Agent process. |
+| agent.pid_mode | string | | Turns on sharing between container and the host operating system the PID address space. |
```

Comment:
Maybe we should mention the possible values here. And phrase it in a way that is less bound to containers.
```suggestion
| agent.pid_mode | string | | Controls access to PID namespaces. When set to `host`, the agent will have access to the PID namespace of the host. |
```